### ISSUE TITLE
Crash on World Join - Can't access registry

### ISSUE THREAD

**Issue by fzzyhmstrs** [Abbie5/emi-ores#4]
Hi abbie, getting the following crash from EMI Ores in the modpack we are building rn. 

whole log: 
[message(25).txt](https://github.com/Abbie5/emi-ores/files/14406943/message.25.txt)

```
[21:51:33] [Server thread/ERROR]:
 Couldn't place player in world
io.netty.handler.codec.EncoderException: Failed to encode: Can't access registry ResourceKey[minecraft:root / minecraft:block] Placed Reference{ResourceKey[minecraft:worldgen/configured_feature / terralith:cliffs/basalt/crack_deepslate]=Configured: net.minecraft.world.gen.feature.OreFeature@79a6c868: net.minecraft.world.gen.feature.OreFeatureConfig@5741cead}
    at net.minecraft.network.PacketByteBuf.method_49396(PacketByteBuf.java:159) ~[client-intermediary.jar:?]
    at net.minecraft.util.Util.getResult(Util.java:983) ~[client-intermediary.jar:?]
    at net.minecraft.network.PacketByteBuf.encodeAsJson(PacketByteBuf.java:159) ~[client-intermediary.jar:?]
    at cc.abbie.emi_ores.networking.packet.S2CSendFeaturesPacket.lambda$write$1(S2CSendFeaturesPacket.java:33) ~[emi_ores-0.1+1.20.1.jar:?]
    at net.minecraft.network.PacketByteBuf.method_34064(PacketByteBuf.java:267) ~[client-intermediary.jar:?]
    at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?]
    at net.minecraft.network.PacketByteBuf.writeMap(PacketByteBuf.java:265) ~[client-intermediary.jar:?]
    at cc.abbie.emi_ores.networking.packet.S2CSendFeaturesPacket.write(S2CSendFeaturesPacket.java:30) ~[emi_ores-0.1+1.20.1.jar:?]
    at net.fabricmc.fabric.api.networking.v1.ServerPlayNetworking.send(ServerPlayNetworking.java:442) ~[fabric-networking-api-v1-1.3.11+1802ada577-4657550f9c265bf4.jar:?]
    at cc.abbie.emi_ores.EmiOres.lambda$onInitialize$1(EmiOres.java:27) ~[emi_ores-0.1+1.20.1.jar:?]
    at net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents.lambda$static$8(ServerLifecycleEvents.java:86) ~[fabric-lifecycle-events-v1-2.2.2+1802ada577-d012b44cfa5b8a32.jar:?]
    at net.minecraft.server.PlayerManager.handler$cml000$fabric-lifecycle-events-v1$hookOnPlayerConnect(PlayerManager.java:6021) ~[client-intermediary.jar:?]
    at net.minecraft.server.PlayerManager.onPlayerConnect(PlayerManager.java:219) ~[client-intermediary.jar:?]
    at net.minecraft.server.network.ServerLoginNetworkHandler.addToServer(ServerLoginNetworkHandler.java:131) ~[client-intermediary.jar:?]
    at net.minecraft.server.network.ServerLoginNetworkHandler.acceptPlayer(ServerLoginNetworkHandler.java:118) ~[client-intermediary.jar:?]
    at net.minecraft.server.network.ServerLoginNetworkHandler.redirect$cod000$fabric-networking-api-v1$handlePlayerJoin(ServerLoginNetworkHandler.java:563) ~[client-intermediary.jar:?]
    at net.minecraft.server.network.ServerLoginNetworkHandler.tick(ServerLoginNetworkHandler.java:68) ~[client-intermediary.jar:?]
    at net.minecraft.network.ClientConnection.tick(ClientConnection.java:259) ~[client-intermediary.jar:?]
    at net.minecraft.server.ServerNetworkIo.tick(ServerNetworkIo.java:172) ~[client-intermediary.jar:?]
    at net.minecraft.server.MinecraftServer.tickWorlds(MinecraftServer.java:908) ~[client-intermediary.jar:?]
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:824) ~[client-intermediary.jar:?]
    at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:105) ~[client-intermediary.jar:?]
    at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:671) ~[client-intermediary.jar:?]
    at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:265) ~[client-intermediary.jar:?]
    at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by Abbie5** [comment]
Thanks, I'll have a look at it. It seems similar to #1

**Comment by Abbie5** [comment]
@fzzyhmstrs I have released version v0.2+1.20.1 which has the fix for this issue included, as well as some other changes.
